superset/sql_parse.py

```diff
@@ -222,6 +220,15 @@ def __extract_from_token(self, token: Token):  # pylint: disable=too-many-branch
                     if not self.__is_identifier(token2):
                         self.__extract_from_token(item)
 
+    def __process_token_group(self, tokens):  # flatten nested tokens to 1D array
```

Comment:
`sqlparse` has a `flatten` method and thus you may want to explore whether this serves the same purpose as said function.